src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1822,6 +1823,13 @@ def create_vmss(vmss_name, resource_group_name, image,
                                 else '{}/publicIPAddresses/{}'.format(network_id_template,
                                                                       public_ip_address))
 
+    def _get_public_ip_address_allocation(value, sku):
+        IPAllocationMethod = get_sdk(ResourceType.MGMT_NETWORK, 'IPAllocationMethod', mod='models')
```

Comment:
I did it in the method because there are paths where you won't need to import this at all :)  It will only get called max once anyways.